app/api.ts

```diff
@@ -15,11 +15,12 @@ export type Period = AbsolutePeriod | [from: number]
 
 export type JobID = string
 
+export type DialogInfoMap = Record<string, Omit<DialogInfo, 'id'>>
```

Comment:
I would avoid "Map" in the type name - it's an object (not a `Map`) anyways...should it be? You could `.size` it easier.

Also, I'd use `ToString<EntityID>`(?) for the key to provide more info.